src/main/webapp/scripts/my-trips.js

```diff
@@ -196,23 +196,99 @@ async function findHotel() {
  * @param {Object} json the resulting JS object from calling the
  *                      Places API for the centerpoint.
  */
-function parseAndRenderHotelResults(json) {
+async function parseAndRenderHotelResults(json, centerPoint) {
```

Comment:
What determines if you make a function async or not?

Reply:
> What determines if you make a function async or not?

My explanation for this may be poor but async functions give you the ability to use the await keyword which I live other functions do not. If you need to call on another function of fetch something while you are still in your current function. (This case parseAndRenderHotelResults) then you can use the await keyword to say grab a value or like in this function he grabs a photo. Then your function continues. I don't believe this is possible with making it an async function.

Reply:
I see, that makes sense, thanks again Josh!

Reply:
Yeah, I made some API calls in this function so I had to add `async` since I like to use `async/await` instead of chaining `.then()` calls; I just think it's more readable.